velox/exec/tests/AggregationFuzzer.cpp

```diff
@@ -683,22 +718,20 @@ void AggregationFuzzer::verify(
 
     auto altPlans =
         makeAlternativePlans(groupingKeys, aggregates, masks, input);
-
-    for (const auto& altPlan : altPlans) {
-      auto altResultOrError = execute(altPlan);
-
-      // Compare results or exceptions (if any). Fail is anything is different.
-      if (resultOrError.exceptionPtr || altResultOrError.exceptionPtr) {
-        // Throws in case exceptions are not compatible.
-        velox::test::compareExceptions(
-            resultOrError.exceptionPtr, altResultOrError.exceptionPtr);
-      } else if (!orderDependent) {
-        VELOX_CHECK(
-            assertEqualResults(
-                {resultOrError.result}, {altResultOrError.result}),
-            "Logically equivalent plans produced different results");
-      }
+    testPlans(altPlans, !orderDependent, resultOrError);
+
+    // Evaluate same plans on flat inputs.
+    std::vector<RowVectorPtr> flatInput;
+    for (const auto& vector : input) {
+      auto flat = BaseVector::create<RowVector>(
+          vector->type(), vector->size(), vector->pool());
+      flat->copy(vector.get(), 0, 0, vector->size());
+      flatInput.push_back(flat);
```

Comment:
Where is flatInput used?

Reply:
Oops. It should be used in the following call to makeAlternativePlans. Let me fix.